tests/config/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -0,0 +1,224 @@
+// Vitest Snapshot v1
+
+exports[`basic.styl 1`] = `
+a, div {
+  color: white 
+  span {
+    background rgba(0,0,0,1)
+    color #ff00e3;
+    border #9C9C9C  2rpx solid
+  }
+}
+
+.cl {
+  display: flex}~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+a
+div {
+  color: white;
+  span {
+    background: rgba(0, 0, 0, 1);
+    color: #ff00e3;
+    border: #9C9C9C 2rpx solid;
+  }
+}
+
+.cl {
+  display: flex;
+}
+
+`;
+
+exports[`basic.vue 1`] = `
+<template>
+    <div></div>
+</template>
+
+<style lang="stylus">
+
+
+font-size = 14px
+
+body
+   font font-size Arial, sans-serif
+   text-color = white
+   
+   p
+    color text-color
+
+
+#logo
+   position: absolute
+   top: 50%
+   left: 50%
+   width: w = 150px
+   height: h = 80px
+   margin-left: -(w / 2)
+   margin-top: -(h / 2)
+</style>~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+<template>
+  <div></div>
+</template>
+
+<style lang="stylus">
+font-size = 14px
+
+body {
+  font: font-size Arial, sans-serif;
+  text-color = white
+
+  p {
+    color: text-color;
+
+
+  }
```

Comment:
The blank lines here are weird. In fact with the right curly bracket present, we don't need these extra blank lines to separate declarations. (Well it's still preferable if we have **one** blank line between declaration blocks.)